### DISCUSSION TITLE
Drop your ideas here for my repository and contributors.

### DISCUSSION THREAD

**Discussion by Reptora7TB** [Reptora7TB/Reptora7TB#10] [General]
Drop your ideas here for my repository and contributors.

**Comment by JordyzFlex** [comment]
well, how about you @Reptora7TB make your own website and add the readme.md list

**Reply by Reptora7TB** [reply]
it's not bad idea.

**Reply by Hagataii** [reply]
how to make a website on github?

**Reply by JordyzFlex** [reply]
> how to make a website on github?

http://pages.github.com/

**Reply by Hagataii** [reply]
thanks.
and bye!

**Reply by JordyzFlex** [reply]
> it's not bad idea.

it's my work :P
